src/runtime/manifest-primitive-parser/smoke/smoke.html

```diff
@@ -0,0 +1,22 @@
+<style>
+  body {
+    font-family: sans-serif;
+  }
+</style>
+
+<pre id="output"></pre>
+
+<script type="module">
+  import {Loader} from '../../../../build/platform/loader.js'
+  import {ManifestPrimitiveParser} from '../../../../build/runtime/manifest-primitive-parser/manifest-primitive-parser.js';
+  const go = async () => {
+    const loader = new Loader({'https://$particles': `../../../../particles`});
+    const path = 'https://$particles/Dataflow/OverviewAction.arcs';
+    const ast = await ManifestPrimitiveParser.load(path, loader, {filename: path});
+    window.output.innerText = JSON.stringify(ast, (k,v) => k === 'location' ? undefined : v, '  ');
+    console.log(`Ast has ${ManifestPrimitiveParser.extract('recipe', ast).length} recipe(s)`);
+    console.log(`Ast has ${ManifestPrimitiveParser.extract('store', ast).length} store(s)`);
+    console.log(`Ast has ${ManifestPrimitiveParser.extract('import', ast).length} import(s)`);
```

Comment:
If these results are useful for verifying the parser output, they should be in a unit test.